### PR TITLE
is_valid changes for testing create_srp

### PR DESCRIPTION
**Pull request by AKktk**
Final edits to is_valid

### CODE REVIEW

File: R/srp_model.R [hunk 1]
```diff
@@ -71,8 +71,9 @@ create_srp_model <- function(
 #' @importFrom checkmate assert_vector
 is_valid.srp_model <- function(mdl) { # nolint
   with(mdl, {
-    stopifnot(logodds_mean[1] < logodds_max[1])
-    stopifnot(logodds_mean[2] < logodds_max[2])
+    checkmate::assertTRUE(logodds_mean[length(attr(mdl, "group_id"))] <
```
Comment (kkmann) [inline R/srp_model.R]: hm, not sure this is correct. why not `all(logodds_mean < logodds_max)`  ?

File: tests/testthat/test-srp_model.R [hunk 2]
```diff
@@ -37,19 +37,19 @@ test_that("private function is_valid throws correct errors", {
   expect_error(
```
Comment (kkmann) [inline tests/testthat/test-srp_model.R:39]: we need a test case showing that a single-group design can be instantiated. that was the problem in the first place, right?

**Review by kkmann — CHANGES_REQUESTED** [review]
Hey, thanks hope you can address the two points. Kudos for sticking to the style guides :)

File: tests/testthat/test-srp_model.R (hunk 2 above)
Comment (AKktk) [reply tests/testthat/test-srp_model.R:39]: I tried single-group, like we did. It works

File: R/srp_model.R (hunk 1 above)
Comment (AKktk) [reply R/srp_model.R]: I tried that. But it always gives error if there are more than 1 groups.
It considers only 1 logical variable as input i think

Comment (AKktk) [reply R/srp_model.R]: can solve it with sum

Comment (AKktk) [reply R/srp_model.R]: but one another test case fails if I use this method.

**Review by kkmann — APPROVED** [review]

**Pull request merged**